weather/templates/weather/partials/daily-summary-list-item.html

```diff
@@ -0,0 +1,98 @@
+
+            {% if alerts|length > 0 %}
+                <div class="grid-col-12 tablet:grid-col-6 tablet:display-flex tablet:flex-justify-end margin-top-2 tablet:margin-top-0 daily-alert-summary">
+                    {% summary_alert_link alerts=alerts %}
+                </div>
+            {% endif %}
+        </div>
+    </div>
+    <div class="grid-row grid-gap">
+        {% for period in day.periods %}
+            <div class="wx-daily-forecast-summary-area wx-daily-forecast-summary-area--{% if period.isOvernight %}overnight{% elif period.isDaytime%}daytime{%  else %}other{% endif %}">
+                <span class="display-block font-family-mono text-uppercase font-body-xs margin-bottom-1 text-primary-darker wx-marker-before">
+                    {{label}}
+                    {% if period.isOvernight %}
+                        {% trans "daily-forecast.labels.overnight.01" %}
+                    {% elif period.isDaytime %}
+                        {% trans "daily-forecast.labels.day.01" %}
+                    {% else %}
+                        {% trans "daily-forecast.labels.night.01" %}
+                    {% endif %}
+                    {% if period.timeLabel %}
+                        <span class="daily-period-time-label text-gray-50">
+                            ({{period.timeLabel}})
+                        </span>
+                    {% endif %}
+                </span>
+                <div class="display-flex flex-row flex-align-start">
+                    {% if period.data.icon.base %}
+                        <div class="width-5 height-5 margin-right-105">
+                            <svg role="img" aria-hidden="true" class="width-full height-full">
+                                <use xlink:href="/assets/images/spritesheet.svg#{{period.data.icon.base }}"></use>
+                            </svg>
+                        </div>
+                    {% endif %}
+                <div class="flex-1">
+                    <p class="daily-short-forecast font-body-sm margin-top-2px margin-bottom-0 marker-height-sans-3">
+                        {{period.data.description}}
+                    </p>
+                    {% if period.data.probabilityOfPrecipitation.hourlyMax and period.data.probabilityOfPrecipitation.hourlyMax > 1 %}
+                        <p class="text-gray-50 font-body-xs margin-top-05 margin-bottom-0">
+                            {% trans_with_args "daily-forecast.text.chance-precip.02" percent=period.data.probabilityOfPrecipitation.hourlyMax %}
+                        </p>
+                    {% endif %}
+                </div>
+            </div>
+        </div>
+        {% endfor %}
+    </div>
+
+    {% if day.hours|length > 0 %}
+    <div>
+        {#  Summary Temperature Chart #}
+        <div class="wx-chart-wrapper margin-top-0 margin-bottom-4">
+            <span class="font-family-mono text-primary-dark usa-sr-only">
+                {{ "hourly-table.labels.temperature.01" | t }}
```

Comment:
There's a few of these `| t`. Are these holdovers or is this a different way to do translations than `trans` /. `trans_with_args`?

Reply:
Yes these are holdovers. Additionally, there are places where we cannot use the `trans` tag because the argument to some other tag itself needs to be translated -- and I believe that can only be done with a filter. Btw the `t` filter is defined in our custom `weather_i18n` template tags file